mta/README.md

```diff
@@ -0,0 +1,47 @@
+# MTA - migration analysis workflow
+
+# Synopsis
+This workflow is an assessment workflow type, that invokes an application analysis workflow using MTA[1]
+and optionally returns the [move2kube](./m2k) workflow reference, to run next if the analysis is considered to be successful.
```

Comment:
'Optionally' is misleading. The user can't currently choose whether to return the M2K workflow or not - it's linked in the Results no matter what.

Reply:
Ack. will update